classes/interaction.lua

```diff
@@ -29,6 +29,13 @@ function Interaction:constructor(data)
     self.options = data.options
     self.isDestroyed = false
     self.DuiOptions = {}
+    self.sprite = data.sprite
+
+    if data.sprite ~= nil then
+        pcall(function()
```

Comment:
generally with pcall its pcall(functionname, args...)

```lua
pcall(lib.requestStreamedTextureDict, data.sprite.dict)
```

Reply:
Fixed in commit [a7c80b5](https://github.com/Sleepless-Development/sleepless_interact/pull/7/commits/a7c80b54e229ed6e36778588a8bdf2fd65edf587)